pkg/kubelet/util.go

```diff
@@ -40,6 +40,16 @@ func CapacityFromMachineInfo(info *cadvisorApi.MachineInfo) api.ResourceList {
 
 // Check whether we have the capabilities to run the specified pod.
 func canRunPod(pod *api.Pod) error {
+	if pod.Spec.HostIPC {
```

Comment:
should be a test checking this allows/denies properly based on the configured HostIPCSources